src/doc_handler.py

```
@@ -0,0 +1,526 @@
+        
+        # Extract text from each attachment
+        for attachment in attachments:
+            attachment_name = attachment.get("name", "unknown")
+            extracted_text = extract_text_from_attachment(
+                attachment, access_token, message_id, email_address, base_url
+            )
+            
+            if extracted_text:
+                results[attachment_name] = extracted_text
+        
+        return results
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)